src/XRP/shared/rippled-error-messages.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * A class for creating constants that represent possible error messages received from the rippled JSON RPC.
+ */
+export default class RippledErrorMessages {
+  static accountNotFound = 'actNotFound'
+  static invalidExcludedAddress = 'invalidHotwallet'
```

Comment:
this seems ok but i think it would be fine also to have an enum for these

Reply:
ah gotcha didn't think typescript supported string literal enums but it does